public_html/css/index.css

```diff
@@ -154,3 +154,8 @@ h1 {
     /*border: 1px solid green;*/
     display: inline-block;
 }
+
+.status-div--reviewer {
+    margin-top: 10px;
+    white-space: nowrap;
+}
```

Comment:
A smaller font would avoid the username from potentially going offscreen, since the `nowrap` keeps it from wrapping. I added that rule because for most usernames it's going to make it look nicer (`MusikAnimal (WMF)` shows on two lines rather than one), but I was thinking we might also have an ellipsis; e.g. `Reviewed by ReallyLongUsernameBlah...` 

I definitely prefer CSS over allowing the browser to style things, but here since we're using Bootstrap it's applying explicit sizing with CSS, so we might consider using `<small>` to be acceptable.